allensdk/brain_observatory/behavior/behavior_ophys_api/behavior_ophys_nwb_api.py

```diff
@@ -167,3 +179,28 @@ def get_task_parameters(self) -> dict:
             data['response_window'] = OphysBehaviorTaskParametersSchema().load({'response_window': data['response_window']}, partial=True)['response_window']
             data['blank_duration'] = OphysBehaviorTaskParametersSchema().load({'blank_duration': data['blank_duration']}, partial=True)['blank_duration']
         return data
+
+    def get_cell_specimen_table(self) -> pd.DataFrame:
+        df = self.nwbfile.modules['two_photon_imaging'].data_interfaces['image_segmentation'].plane_segmentations['cell_specimen_table'].to_dataframe()
+        df.index.rename('cell_roi_id', inplace=True)
+        df['cell_specimen_id'] = [None if csid == -1 else csid for csid in df['cell_specimen_id'].values]
+        df['image_mask'] = [mask.astype(bool) for mask in df['image_mask'].values]
+        return df
+
+    def get_dff_traces(self) -> pd.DataFrame:
+        dff_nwb = self.nwbfile.modules['two_photon_imaging'].data_interfaces['dff'].roi_response_series['traces']
+        return pd.DataFrame({'dff': [x for x in dff_nwb.data[:]]},
+                             index=pd.Index(data=dff_nwb.rois.table.id[:], name='cell_roi_id'))
+
+    def get_corrected_fluorescence_traces(self) -> pd.DataFrame:
+        corrected_fluorescence_nwb = self.nwbfile.modules['two_photon_imaging'].data_interfaces['corrected_fluorescence'].roi_response_series['traces']
+        return pd.DataFrame({'corrected_fluorescence': [x for x in corrected_fluorescence_nwb.data[:]]},
+                             index=pd.Index(data=corrected_fluorescence_nwb.rois.table.id[:], name='cell_roi_id'))
+
+    def get_motion_correction(self) -> pd.DataFrame:
+
+        motion_correction_data = {}
+        motion_correction_data['x'] = self.nwbfile.modules['motion_correction'].get_data_interface('x').data[:]
```

Comment:
might not be the right time/place for this comment, but it seems like at some point we ought to name x and y something descriptive (row and column even). Since this nomenclature is embedded in LIMS, changing it is nontrivial, but it might be a good thing to rename on the way out of or into the nwb file.

Reply:
Agreed.  I had the same thought; I am going to leave it as-is for now, so that the data values can be more easily human-associated with their sources in LIMS, and rethink this decision if we ever find these names confusing for downstream processing (relative to the NWB files)